bookwyrm/templates/guided_tour/home.html

```diff
@@ -0,0 +1,216 @@
+                }
+            ],
+    }
+])
+
+const homeTour = new Shepherd.Tour({
+        exitOnEsc: true,
+    });
+
+homeTour.addSteps([
+    {
+        text: "{% trans 'Search for books, users, or lists using this search box.' %}",
+        title: "{% trans 'Search box' %}",
+        attachTo: {
+            element: "#tour-search",
+            on: "bottom",
+        },
+        highlightClass: 'tour-element-highlight',
+        buttons: [
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'Search book records by scanning an ISBN barcode using your camera.' %}",
+        title: "{% trans 'Barcode reader' %}",
+        attachTo: {
+            element: "#tour-barcode",
+            on: "bottom",
+        },
+        buttons: [
+            {
+                action() {
+                    return this.back();
+                },
+                secondary: true,
+                text: "{% trans 'Back' %}",
+            },
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'Use these links to discover the latest news from your feed, lists of books by topic, and the latest happenings on this Bookwyrm server!' %}",
+        title: "{% trans 'Navigation Bar' %}",
+        attachTo: {
+            element: "#tour-navbar-start",
+            on: "bottom",
+        },
+        highlightClass: 'tour-element-highlight',
+        buttons: [
+            {
+                action() {
+                    return this.back();
+                },
+                secondary: true,
+                text: "{% trans 'Back' %}",
+            },
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'Books on your reading status shelves will be shown here.' %}",
+        title: "{% trans 'Your Books' %}",
+        attachTo: {
+            element: "#tour-suggested-books",
+            on: "right",
+        },
+        highlightClass: 'tour-element-highlight',
+        buttons: [
+        {
+                action() {
+                    return this.back();
+                },
+                secondary: true,
+                text: "{% trans 'Back' %}",
+            },
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'Updates from people you are following will appear on your timeline here.<br><br>The <strong>Books</strong> tab shows just the updates about reading activity.' %}",
```

Comment:
I would change the description of the books tab here, because as it is I think it makes it sound like you see a subset of your home timeline, rather than updates from anyone posting about your books.

Maybe something like "The **Books** tab shows you updates from anyone about your books"?

Reply:
To be honest I was struggling to understand what the Books tab is for - I didn't realise it was for "your books", though not you've pointed that out it's rather obvious. Thanks for clarifying!

Reply:
The tour is doing it's work already :)